engine/script/src/test/test_script_sys.cpp

```diff
@@ -92,7 +92,6 @@ bool RunString(lua_State* L, const char* script)
 }
 
 
-#if defined(__NX__) && !defined(GITHUB_CI) // we cannot run this test unless we have a device connected
```

Comment:
@JCash I'm a bit confused by this. Why don't we want to run these tests for all platforms?

Reply:
Not sure.
Possibly something I had to disable at the time